src/coreclr/tests/src/profiler/native/eventpipeprofiler/eventpipeprofiler.cpp

```diff
@@ -24,7 +24,7 @@ HRESULT EventPipeProfiler::Initialize(IUnknown* pICorProfilerInfoUnk)
     }
 
     // No event mask, just calling the EventPipe APIs.
-    if (FAILED(hr = _pCorProfilerInfo12->SetEventMask2(COR_PRF_MONITOR_JIT_COMPILATION, 0)))
+    if (FAILED(hr = _pCorProfilerInfo12->SetEventMask2(COR_PRF_MONITOR_JIT_COMPILATION | COR_PRF_DISABLE_ALL_NGEN_IMAGES, 0)))
```

Comment:
I would comment all of these and detail why the DISABLE_ALL_NGEN_IMAGES flag has been set. Otherwise someone might assume it is a requirement for using the feature rather than a requirement due to a specific independent implementation choice in the test code.